interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer:the command this functions as a way to identify variables made within the current component.
 
-  Researched answer:
+  Researched answer: by how `this` is used `this` comunicates you are refering to an object. It can refer to an individual object when used in an object method or global object if used alone.
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: react is one of afew ways to provide functionality as well as utility in html
 
-  Researched answer:
+  Researched answer: React allows a webpage to update without requireing a full reload of the page.
```

Comment:
The hot reload is definitely a nice feature. Checkout what the syllabus is saying about React. The major use is creating user interfaces.